modulus/distributed/utils.py

```diff
@@ -198,7 +218,7 @@ def _split(input_, dim_, group=None):  # pragma: no cover
     return output
 
 
-def _gather(input_, dim_, group=None):  # pragma: no cover
+def _gather(input_, dim_, shapes_=None, group=None):  # pragma: no cover
```

Comment:
Is there something `all_gather_v_wrapper` is missing? Or is there another reason to keep kind of duplicated codes going forward?